integration-tests/package.json

```diff
@@ -14,7 +14,7 @@
   "license": "ISC",
   "dependencies": {
     "@polkadot/api": "^8.6.2",
-    "@types/jest": "^27.5.0",
+    "@types/jest": "^28.1.0",
```

Comment:
@CAGS295, @jfrank-summit while on the topic of dependencies: shouldn't the typescript definitions be the same version as the main package itself?  ATM jest is at `28.0.3` while upstream there is already `28.1.0` but apparently Dependenbot didn't pick it up yet.

Reply:
I don't know if a version mismatch here will cause problems or how severe they may be.